sources/vega/events/v1/events.proto

```diff
@@ -12,7 +12,31 @@ import "vega/oracles/v1/data.proto";
 import "vega/commands/v1/commands.proto";
 import "vega/commands/v1/validator_commands.proto";
 
-//DelegationBalanceEvent - updates on the delegation balance of a party to a node in the current epoch in effect 
+// StakingEvent - an event notifying of stake being deposited or removed for a given party
+// These events are emitted for every Staking deposit or removed accepted by the network
+message StakingEvent {
+  enum Kind {
+    // Default value
+    KIND_UNSPECIFIED = 0;
+    // Indicate of a stake deposit instruction
+    KIND_DEPOSIT = 1;
+    // Indicate of a stake remove instruction
+    KIND_REMOVE = 2;
```

Comment:
WITHDRAW?

Reply:
Well, the ethereum event have been called Remove, so I matched the naming with it there: https://github.com/vegaprotocol/Staking_Bridge/blob/main/contracts/IStake.sol#L10.

Do you want me to name it withdraw here in the proto? I'm happy to.

Reply:
leave it for now